test/common/testvm.py

```diff
@@ -259,7 +273,8 @@ def _kill_ssh_master(self):
             self.ssh_master = None
         if self.ssh_process:
             self.ssh_process.stdin.close()
-            self.ssh_process.wait()
+            if not self._abort:
+                self.ssh_process.wait()
```

Comment:
If my comment above is true, then killing the ssh master would be appropriate. It's in an unknown state, and should be reestablished as necessary. At least that was the intent when I wrote the code. Let me know if this assumption doesn't hold.